src/tranquility/mini/init.lua

```diff
@@ -0,0 +1,11 @@
+require("tranquility.mini.visitor")
+
+function Parse_mini(code)
+	local raw_ast = Parse(code)
+	return Visitor:visit(raw_ast)
+end
+
+-- function Mini(code)
```

Comment:
Would you please delete this commented out version?

Reply:
The idea looks like this: lpeg => a messy, heavily nested AST => Parse_mini() => a meaningful, unified AST => Mini() => the actual pattern. 
And the Mini() can just be plug into the Reify() and have a working mini-notation.

Reply:
Ah thanks! I'm starting to see the game plan now. We just need a few more basic functions to allow the AST to be parsed into a pattern. We can leave this here as a guide if that's helpful in the mean time.